_posts/en/newsletters/2023-07-19-newsletter.md

```diff
@@ -0,0 +1,164 @@
+  remote signing device instead of local keys. The [beta release][VLS gitlab]
+  supports CLN and LDK, layer-1 and layer-2 validation rules, backup/recovery
+  capabilities, and provides a reference implementation. The [blog
+  post][VLS blog] announcement also calls for testing, feature requests, and feedback from the community.
+
+- **BitGo adds MuSig2 support:**
+  BitGo [announced][bitgo blog] support for [BIP327][] ([MuSig2][topic musig])
+  and noted the reduced fees and additional privacy compared to their other
+  supported address types.
+
+- **Peach adds RBF support:**
+  The [Peach Bitcoin][peach website] mobile application for peer-to-peer
+  exchange [announced][peach tweet] support for [Replace-By-Fee (RBF)][topic rbf] fee bumping.
+
+- **Phoenix wallet adds splicing support:**
+  ACINQ [announced][acinq blog] beta testing for the next version of their
+  Phoenix mobile Lightning wallet. The wallet supports a single dynamic channel
+  that is rebalanced using [splicing][topic splicing] and
+  a mechanism similar to the [swap-in-potentiam][news233 sip] technique (see
+  [Podcast #259][pod259 phoenix]).
+
+- **Mining Development Kit call for feedback:**
+  The team working on the Mining Development Kit (MDK) has [posted an update][MDK blog] on their
+  progress to develop hardware, software, and firmware for Bitcoin mining systems. The post
+  calls for feedback from the community about use cases, scope, and approach.
+
+- **Binance adds Lightning support:**
+  Binance [announced][binance blog] support for sending (withdrawals) and
+  receiving (deposits) using the Lightning Network.
+
+- **Nunchuk adds CPFP support:**
+  Nunchuk [announced][nunchuk blog] support for [Child-Pays-For-Parent
+  (CPFP)][topic cpfp] fee bumping for both senders and receivers of a transaction.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update Tuesday -->
+
+- Core Lightning 23.05.2 FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27411][] prevents a node from advertising its Tor or
+  I2P address to peers on other networks (such as plain IPv4 or IPv6)
+  and won't advertise its address from non-[anonymity networks][topic
+  anonymity networks] to peers on Tor and I2P.  This helps prevent
+  someone from connecting a node's regular network address to one of its
```

Comment:
```suggestion
  someone from linking a node's regular network address to one of its
```
"connecting" could be confusing since there are also network connections in this context.